.github/workflows/tizen_integration_test.yml

```
@@ -0,0 +1,29 @@
+name: Integration test on GBS Tizen/x64 from Ubuntu
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v1
+    - name: prepare deb sources for GBS
+      run: echo "deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_20.04/ /" | sudo tee /etc/apt/sources.list.d/tizen.list
+    - name: install GBS
+      run: sudo apt-get update && sudo apt-get install -y gbs
+    - name: configure GBS
+      run: cp .github/workflows/tizen.gbs.conf ~/.gbs.conf
+    - name: run GBS
+      run: gbs build
+    - name: get nnstreamer-main
+      run: git clone https://github.com/nnstreamer/nnstreamer.git
+    - name: run GBS on nnstreamer with unit test
+      run: pushd nnstreamer && gbs build --define "unit_test 1" && popd
```

How about applying edge-only test?
```suggestion
      run: pushd nnstreamer && gbs build --define "edge_test 1" && popd
```
With this PR, https://github.com/nnstreamer/nnstreamer/pull/3891, we can reduce testing time.